py/scripts/new_version.py

```diff
@@ -0,0 +1,34 @@
+import toml  # type: ignore [import]
+import json
+
+
+def change_parse_dep(data: dict, version: str) -> dict:
+    dep, ver = (
+        data["project"]["dependencies"][0].split(">=")[0],
+        data["project"]["dependencies"][0].split(">=")[1],
+    )
+    ver = version
+    dependency = dep + ">=" + ver
+    data["project"]["dependencies"].insert(0, dependency)
+    data["project"]["dependencies"].pop(1)
+    return data
+
+
+with open("py/package.json", "r") as p:
+    package_data = json.load(p)
+
+with open("py/pyproject.toml", "r") as t:
+    toml_data = toml.load(t)
+
+with open("py/llama_parse/pyproject.toml", "r") as pt:
```

Comment:
llama_parse has it's own pyproject file, so it needs it's own package.json file

Reply:
then the goal of this script is just to sync the version from package.json to pyproject.toml (the package.json version is updated by changeset thinking it's a TS package)